brion/blueConfig.h

```diff
@@ -74,6 +74,12 @@ class BlueConfig : public boost::noncopyable
                                       const std::string& sectionName,
                                       const std::string& key ) const;
 
+    /**
+     * @return the user and circuit file target parsers.
+     * @version 1.7
+     */
+    BRION_API Targets getTargetParsers() const;
```

Comment:
I have to admit that getTargets is clear to me because it returns a vector of brion::Target, I wrote that in the beginning, but then I tried to think with someone else's mind and found it ambiguous (I think I was trying to think as a newbie). Maybe the root of the problem is that Target is not a good choice to name that concept, but I'll do as you ask.